File: ghc6/TestTest.hs
module TestTest where

 import TestEnv

 tests te =
  return te >>=
  checkComplainAndAdjustExpected 0 >>=
  test True >>=
  test True >>=
  checkComplainAndAdjustExpected 2 >>=
  test True >>=
  test True >>=
  test True >>=
  checkComplainAndAdjustExpected 3 >>=
  return

File: mono-2-vbnc/TestTest.vb
Namespace TestTest

    Module TestTest

        Sub Tests( ByRef test As Test)

            test.CheckComplainAndAdjustExpected(0)
            test.Test(True)
            test.Test(True)
            test.CheckComplainAndAdjustExpected(2)
            test.Test(True)
            test.Test(True)
            test.Test(True)
            test.CheckComplainAndAdjustExpected(3)

        End Sub
    End Module
End Namespace

